scripts/templates/helper.py

```diff
@@ -695,6 +695,14 @@ def make_param_lines(namespace, tags, obj, py=False, decl=False, meta=None, form
         name = _get_param_name(namespace, tags, item)
         if py:
             tname = get_ctype_name(namespace, tags, item)
+            # Handle fptr_typedef
+            if "fptr_typedef" in meta:
+                fptr_types = list(meta['fptr_typedef'].keys())
```

Comment:
Could you clarify what is happening here.

For each parameter `item` we construct a list of all function pointer types that the function takes as arguments, we then make the substitution to the c typename for each of these function pointer arguments, then we check if the substituted typename for `item` is equal to any of those substituted function pointer types and if it is we set it to `c_void_p`?